Dockerfile

```diff
@@ -42,16 +49,16 @@ ENV PATH="/home/${NB_USER}/.local/bin:${PATH}"
 
 # COPY requirements/dev.txt requirements-dev.txt
 COPY requirements/base.txt requirements-base.txt
-COPY prepline_${PIPELINE_PACKAGE}/ prepline_${PIPELINE_PACKAGE}/
-COPY exploration-notebooks exploration-notebooks
-COPY pipeline-notebooks pipeline-notebooks
-
 RUN python3.8 -m pip install pip==${PIP_VERSION} \
   && pip3.8 install  --no-cache  -r requirements-base.txt \
   && pip3.8 install --no-cache "detectron2@git+https://github.com/facebookresearch/detectron2.git@v0.6#egg=detectron2"
 
+COPY prepline_${PIPELINE_PACKAGE}/ prepline_${PIPELINE_PACKAGE}/
+COPY exploration-notebooks exploration-notebooks
+COPY pipeline-notebooks pipeline-notebooks
+
 EXPOSE 5000
 
-ENTRYPOINT ["uvicorn", "unstructured_inference.api:app", \
+ENTRYPOINT ["uvicorn", "prepline_general.api.app:app", \
```

Comment:
This looks good now, how was this working before this change? Or was there a refactor that broke it at some point?

Reply:
The makefile was overriding the entrypoint (note that unstructured-inference is doing this too so small thing to fix at some point).